documentation/asciidoc/computers/camera/rpicam_jpeg.adoc

```diff
@@ -1,50 +1,19 @@
 === `rpicam-jpeg`
 
-`rpicam-jpeg` is a simple still image capture application. It deliberately avoids some of the additional features of `rpicam-still`, which attempts to emulate `raspistill` more fully. This means that the code is significantly easier to understand, while still providing many of the same features.
+`rpicam-jpeg` helps you capture images on Raspberry Pi devices.
```

Comment:
With this introductory paragraph slimmed down so much, I guess a user wouldn't know whether to use `raspicam-still` or `raspicam-jpeg` ? :thinking:

Reply:
`rpicam-still` still keeps this information. But I think `raspi` has been dead long enough that we don't have to clog up our descriptions with constant `raspi` mentions. After all, we have all the libcamera-apps cruft now instead 😉

Reply:
I meant more the fact that `rpicam-jpeg` is a deliberately-simple demo application, designed for code-readability; whereas `rpicam-still` is a more fully-featured application.